cmd/internal/wait.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(waitCmd)
+}
+
+var waitCmd = &cobra.Command{
+	Use:   "wait",
+	Short: "Wait for required service",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		for {
```

Comment:
We can add this of course, bur I would like to understand the reasoning/benefit. This would result in crashing the container, right? And this is one thing this `PR` is  addressing. We  could  of course set the default to `0`, but for what do we need it then?

Reply:
We don't want to avoid it to crash at all costs. I think crashing the init container after a sensible timeout is reached is useful for telling us that something bad happened, instead of waiting forever.

Reply:
We have added a timeout. But I would argue that a bit more determinism would help the cf-operator a lot. Why crash and just try it again. If it works, errors could be hidden. But the timeout should not hurt for sure.

Reply:
I think the main advantage of having the timeout is that it tells us that something went wrong rather than never telling us anything.

Reply:
Debatable :) If it doesn't start at all, you would notice. I would agree if it would not just  be restarted. I am not a big fan of "eventually consistent". We see it from time to time that the `cf-operator` behaves strange (e.g. not creating `StatefulSets` for 1 hour), but  if we keep the system running long enough it turn out to be up and  running.
Which has of course also a good side to it, but I wonder sometime how  indeterministic the `cf-operator` can be.